aggregator/src/main/resources/application.properties

```
@@ -34,6 +34,9 @@ quarkus.log.sentry=false
 quarkus.log.sentry.in-app-packages=*
 quarkus.log.sentry.dsn=FILL_ME
 
+prometheus.pushgateway.url=${PROMETHEUS_PUSH_GATEWAY}
+%test.prometheus.pushgateway.url=localhost:8080
```

I would be simpler if `PROMETHEUS_PUSH_GATEWAY`, `PROMETHEUS_PUSHGATEWAY` and `prometheus.pushgateway.url` were a single thing.

The properties should only contain something like this (key to be confirmed):
```suggestion
prometheus.pushgateway.url=localhost:8080
```
And then the Clowder template could override the value with the `PROMETHEUS_PUSHGATEWAY_URL` env var.